docs/install/knative-with-operators.md

```diff
@@ -94,9 +120,60 @@ kind: KnativeServing
 metadata:
   name: knative-serving
   namespace: knative-serving
-EOF
+spec:
+  version: {{spec.version}}
+  manifests:
+    - URL: https://github.com/knative/serving/releases/download/v${VERSION}/serving-core.yaml
+    - URL: https://github.com/knative/serving/releases/download/v{VERSION}/serving-hpa.yaml
+    - URL: https://github.com/knative/serving/releases/download/v{VERSION}/serving-post-install-jobs.yaml
+    - URL: https://github.com/knative/net-istio/releases/download/v0.17.0/net-istio.yaml
```

Comment:
Strange not to have a {VERSION} here, but I'm assuming it's because there might be a different patch version.

Reply:
For a future update, we should figure out how to unify these and give some additional hints on how versioning should work.